endpoints/test/api_config_manager_test.py

```diff
@@ -205,6 +216,28 @@ def test_lookup_rest_method_with_colon_in_param(self):
     self.assertEqual('guestbook_api.get', method_name)
     self.assertEqual(fake_method, method_spec)
 
+  def test_lookup_rest_method_with_colon_after_param(self):
+    fake_method = {'httpMethod': 'GET',
+                   'path': 'greetings/{id}:hello'}
+    self.config_manager._save_rest_method('guestbook_api.get', 'guestbook_api',
+                                          'v1', fake_method)
+
+    method_name, method_spec, _ = self.config_manager.lookup_rest_method(
+        'guestbook_api/v1/greetings/1:hello', 'GET')
+    self.assertEqual('guestbook_api.get', method_name)
+    self.assertEqual(fake_method, method_spec)
```

Comment:
Should there be a test for a method definition in a path component other that last, eg. "/foo:bar/baz"?
My understanding is that this should be an error.

Reply:
Sure, I'll write one.

Reply:
It actually does work at this stage. However, it fails at the api config stage, and there's a test for that lower down in this PR (in api_config_test.py).